common/changes/@microsoft/api-extractor/path-mapping_2022-04-06-02-38.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/api-extractor",
+      "comment": "Bug fix to API Extractor to properly process projects with path mappings in their tsconfig.",
+      "type": "patch"
```

```suggestion
      "type": "minor"
```

The logic changes to ExportAnalyzer.ts are nontrivial.